test/CMakeLists.txt

```
@@ -32,6 +32,7 @@ IF(ENABLE_LIBWTTEST)
     utils/Base64Test.C
     utils/EraseWord.C
     utils/ParseNumber.C
+    widgets/WTableTest.C
     widgets/WSpinBoxTest.C
```

```suggestion
    widgets/WSpinBoxTest.C
    widgets/WTableTest.C
```

T comes after S in the alphabet.